src/webapp/components/survey/hook/useSurveyForm.ts

```diff
@@ -21,6 +22,17 @@ export function useSurveyForm(formType: SURVEY_FORM_TYPES, eventId: string | und
     } = useCurrentSurveys();
     const [error, setError] = useState<string>();
 
+    const shouldDisableSave = (): boolean => {
```

Comment:
Unless there is a specific idea behind, typically we return the boolean here, not a function with no arguments.